serenity-app/src/main/res/xml/manual_server_preferences.xml

```diff
@@ -23,6 +23,16 @@
         android:key="serverport"
         android:summary="@string/default_port_summary"
         android:title="@string/server_port"/>
+
```

Comment:
We probably want to enable this on the login screen when the user is selected if the user is returned by Emby saying they require a username and password to access the server.  This won't work for multiple profiles unfortunately.

I'll create a issue and refer back to this pr for reference.  If get around to implementing I'll make sure your credited as the author.